examples/composition-api/basic/pages/index.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="container">
+    <button @click="increment">
+      Increment
+    </button>
+    <p>Counter : {{ counter }}</p>
+    <button @click="decrement">
+      Decrement
+    </button>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import VueCompositionApi, { defineComponent, ref } from '@vue/composition-api'
+
+Vue.use(VueCompositionApi)
```

Comment:
Installation of `VueCompositionApi` is done by the Nuxt plugin (`plugins/composition-api.js`)
```suggestion
import { defineComponent, ref } from '@vue/composition-api'
```